src/region.js

```diff
@@ -95,6 +87,49 @@ _.extend(Region.prototype, CommonMixin, {
     return this;
   },
 
+  // https://jsfiddle.net/marionettejs/rm5gnxL4/1/
+  _setElement() {
+    if (!this.el) {
+      throw new MarionetteError({
+        name: classErrorName,
+        message: 'An "el" must be specified for a region.',
+        url: 'marionette.region.html#additional-options'
+      });
+    }
+
+    // Check if `$el` is already correct
+    if (this.$el && this.$el[0] === this.el) { return; }
+
+    this.$el = this.getEl(this.el);
+
+    if(this.$el.length) {
```

Comment:
I believe ideally in a future release of Mn, Regions should no longer query the DOM and we should require that region instantiation and setElement include existing DOM elements.  querying the DOM requires awkward `parentEl` context or it can produce unwanted results that are difficult to track down.  View's should query the DOM and pass the element instead of the selector string.  This would also allow us to instantiate regions at the moment we need them instead of prior to anyone accessing or using them.